app/jobs/activity_tracking_job.rb

```diff
@@ -0,0 +1,9 @@
+class ActivityTrackingJob < ApplicationJob
+  include ActivityTracking
```

Comment:
is this the controller concern? It's not immediately clear here.

I'd duplicate the code here and eventually remove the controller concern.

Something we need to think about is how to "silence" tracking, otherwise we're going to end up with a bunch of activities when we don't mean to (rails console, specs, etc.).